2 - RTB/Documentazione interna/Verbali/23-12-23/23-12-23.typ

```diff
@@ -0,0 +1,74 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "23/12/23",
+  subTitle: "Meeting di retrospettiva e pianificazione",
+  docType: "verbale",
+  authors: (
+    "Gardin Giovanni",
+  ),
+  missingMembers: (
+    "Banzato Alessio",
+  ),
+  timeStart: "15:00",
```

Comment:
compilando il file l'ora e la data viene visualizzata in modo non inteso, credo
<img width="150" alt="Immagine 2023-12-30 201850" src="https://github.com/Error-418-SWE/Documenti/assets/91839513/f0542216-6e49-4d53-8946-5e7da3150de9">

c'è quel @ che non credo sia voluto, chiudi pure la pr, se invece è inteso

Reply:
Sì sì è voluto

Reply:
perfetto mi ero perso questa cosa del template